src/LanguageServer/Protocol/Protocol/LanguageServer.Protocol.csproj

```diff
@@ -1,59 +0,0 @@
-﻿<Project>
```

Comment:
this was an unused project file sitting in the repo (probably from when these types got moved from internal)